src/web_parser.py

```diff
@@ -0,0 +1,73 @@
+# Log parser to create json from a log file
+
+import itertools
+import re
+import json
+
+file_path = "../test_logs/sixrobotlog/LOG_Dolphin0_1_12_2020_____13_01_34/LOG_Dolphin0_1_12_2020_____13_01_34.alog"
+opened_file = open(file_path, "r")
+
+
+# Parameter is an opened file, rather than a file path
+def web_parser(file):
+    # Check to see what type of log file this is, and set log_type and robot_id appropriately
+    if "LOG_Narwhal" in file.name:
+        print("This is a Narwhal log")
+        log_type = "Narwhal"
+        robot_id = "Narwhal"
+    elif "LOG_Dolphin" in file_path:
```

Comment:
Is `file_path` used here on purpose or did you mean to use `file.name`

Reply:
Nope this was a mistake. Good catch.